test/filestorage.js

```diff
@@ -5,7 +5,7 @@ const { FileStorage } = require('..');
 const metatests = require('metatests');
 const common = require('metarhia-common');
 
-const storage = new FileStorage('./root-directory');
+const storage = new FileStorage('./test/fs-root');
```

Comment:
I don't think this is a correct path, maybe './fs-root'?

Reply:
You stated [here](https://github.com/metarhia/filestorage/pull/1#discussion_r228706741):
> If this is for testing it should be `test/fs-root` I think.

so I changed it.

Reply:
But the file is already in the `test` folder, therefore this will actually reference `test/test/fs-root`, wouldn't it?

Reply:
No, it is `test/fs-root`.

Reply:
Oh, I understand now, this is dependent on a directory you are in.
I'd suggest to use `path.join(__dirname, 'fs-root')` in a test instead.